lib/ansible/modules/network/fortios/fortios_ipv4_policy.py

```
@@ -0,0 +1,324 @@
+        state                     = dict(choices=['present', 'absent'], default='present'),
+        src_addr                  = dict(required=True, type='list'),
+        dst_addr                  = dict(required=True, type='list'),
+        src_addr_negate           = dict(type='bool', default=False),
+        dst_addr_negate           = dict(type='bool', default=False),
+        action                    = dict(choices=['accept','deny'], required=True),
+        service                   = dict(aliases=['services'], required=True, type='list'),
+        service_negate            = dict(type='bool', default=False),
+        schedule                  = dict(type='str', default='always'),
+        nat                       = dict(type='bool', default=False),
+        fixedport                 = dict(type='bool', default=False),
+        poolname                  = dict(type='str'),
+        av_profile                = dict(type='str'),
+        webfilter_profile         = dict(type='str'),
+        ips_sensor                = dict(type='str'),
+        application_list          = dict(type='str'),
+    )
+
+    #merge global required_if & argument_spec from module_utils/fortios.py
+    argument_spec.update(fortios_argument_spec)
+    required_if = fortios_required_if
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=required_if,
+    )
+
+    result = dict(changed=False)
+
+    # fail if libs not present
+    msg = ""
+    if not HAS_PYFG:
+        module.fail_json(msg='Could not import the python library pyFG required by this module')
+
+    #define device
+    f = FortiOS( module.params['host'],
+        username=module.params['username'],
+        password=module.params['password'],
+        timeout=module.params['username'],
+        vdom=module.params['vdom'])
+
+    path = 'firewall policy'
+
+    #connect
+    try:
+        f.open()
+    except:
```

Avoid to catch all exceptions ([see](https://docs.python.org/2/howto/doanddont.html#except)), use at least `Exception` (the same applies below).